scenarios/sqs_flag_shop/terraform/source/flask/templates/index.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Item purchase page</title>
+    <link rel="stylesheet" href="../static/index.css">
+</head>
+<body>
+<!--  https://github.com/BoB12-C-G-V/FLAG-Shop -->
```

Comment:
This also should probably point to CloudGoat source code since the Lambda function code is changing